phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -1654,7 +1655,8 @@ else if (linkType == PHYSICAL_TABLE) {
                 else {
                     byte[] maxLookbackAgeInBytes = result.getValue(TABLE_FAMILY_BYTES, MAX_LOOKBACK_AGE_BYTES);
                     if (maxLookbackAgeInBytes != null) {
-                        return PLong.INSTANCE.getCodec().decodeLong(maxLookbackAgeInBytes, 0, SortOrder.getDefault());
+                        return PInteger.INSTANCE.getCodec().decodeInt(maxLookbackAgeInBytes, 0,
```

Comment:
The test in MaxLookbackIT seems to be failing as MAX_LOOKBACK_AGE is of type BIGINT in SYSCAT while we are decoding it via PInteger decoder instead of PLong decoder. I think we need to change the type of MAX_LOOKBACK_AGE column in SYSCAT to INTERGER. 
I am also checking other places where we might we using max lookback as long and will update on comment. Thanks

Reply:
That is great! Thank you very much! I will update the PR accordingly.